pages/overview/overview.py

```diff
@@ -9,7 +9,7 @@
 # disable and re-enable formatter
 # fmt: off
 from .visualizations.active_drifting_contributors import gc_active_drifting_contributors
-from .visualizations.total_contributor_growth import gc_total_contributor_growth
+from .visualizations.new_contributor_growth import gc_new_contributor_growth
```

Comment:
I don't actually like the name "new_contributor_growth" because the new contributors aren't growing, it's that there are more new contributors (tiny semantic difference). could rename to "new_contributors"

Reply:
bet ill change that works for me